tests/components/luftdaten/test_config_flow.py

```diff
@@ -60,6 +60,21 @@ async def test_communication_error(hass: HomeAssistant) -> None:
     assert result2.get("step_id") == SOURCE_USER
     assert result2.get("errors") == {CONF_SENSOR_ID: "cannot_connect"}
 
+    with patch("luftdaten.Luftdaten.get_data", return_value=True), patch(
```

Comment:
`get_data` doesn't return a boolean.
```suggestion
    with patch("luftdaten.Luftdaten.get_data"), patch(
```